index.html.in

```diff
@@ -191,9 +191,9 @@
                                           <i class="fa fa-file-text"></i> <span data-text="Translate_Document">Translate a document</span>
                                       </button>
                                   </div>
-                                  <div class="col-lg-3 col-md-6  pull-right">
+                                  <div class="col-lg-3 col-md-6 pull-right" id="translateOptionsContainer">
```

Comment:
So, I thought about this a bit and I'm not actually a huge fan of the siblings solution. It's clever and is probably the minimal diff or something but it requires carefully inspecting the HTML to figure out what things are siblings, etc. I would prefer a more explicit approach, e.g.

hiding/showing `#translateText *:not(#translateOptionsContainer), #translateOptions *:not(#markUnknownContainer)`?